tests/deployment/test_client_server.py

```diff
@@ -255,28 +247,26 @@ def client_server_simulation(x_train, x_test, model, default_configuration):
 
     # Back to the client
 
-    # Decrypt and de-quantize the result
+    # Decrypt, de-quantize and post-processed the result
     y_pred_on_client_quantized = fhemodel_client.deserialize_decrypt(serialized_result)
     y_pred_on_client_dequantized = fhemodel_client.deserialize_decrypt_dequantize(serialized_result)
```

Comment:
deserialize_decrypt_dequantize has postprocessing as well.
maybe we should rename it, it's confusing

Reply:
yes I agree, maybe this could be done along https://github.com/zama-ai/concrete-ml-internal/issues/3045. In any case, I think you could open an issue for this API change proposal and add it to https://github.com/zama-ai/concrete-ml-internal/issues/3995

Reply:
I added 3045 to the API change proposal list